src/components/dialog/dialog.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @function UIDialog
+ * @version 0.0.2
+ * @desc display a Dialog
+ */
+
+export default function UIDialog(userOptions) {
+
+    var DOM,
+        state = {};
+        
+    const defaults = {
+        dialog:         '.js-dialog',
+        openBtn:        '.js-dialog-btn',
+        closeBtn:       '.js-dialog-close-btn',
+        backdrop:       '.js-dialog-backdrop',
+        isModal:        false,
+        showBackdrop:   true
```

Comment:
One thing me and @davislurve  are doing at work is starting all `boolean` variables with *is* or *has* so this one would be something like `hasBackdrop`. How do you feel about that naming convention?